ipwb/__main__.py

```diff
@@ -145,10 +144,18 @@ def checkArgs(argsIn):
     parser.add_argument(
         '-v', '--version', help='Report the version of ipwb', action='version',
         version='InterPlanetary Wayback ' + ipwbVersion)
+    parser.add_argument(
+        '-u', '--update-check',
+        action='store_true',
+        help=('Checks remotely to determine if the installed version '
+              'of ipwb is current')
```

Comment:
I think there is no need to say that it will be done remotely, which is obvious.

```suggestion
        help='Check if an updated version of ipwb is available'
```